### DISCUSSION TITLE
Como configurar o SSL do mapo

### DISCUSSION THREAD

**Discussion by andremp15** [RamonSilva20/mapos#2135] [General]
Bom dia, e que Deus abençoe a todos nesse dia. Tenha uma dúvida de como configurar o SSL do mapos, alguns navegadores principalmente o Chrome fica notificando que o site não é seguro é muito chato, e com a nova atualização do Firefox o outro navegador da Google também tem o mesmo problema se alguém puder ajudar fico muito agradecido obrigado.

**Comment by RamonSilva20** [comment]
Bom dia. Deus abençoe.
Uma forma que acredito ser mais fácil é usar a Cloudflare. 

https://www.youtube.com/watch?v=FTIBckGhhjw&ab_channel=TudoSobreSites
